api/types/session.go

```diff
@@ -534,3 +529,38 @@ func (r *DeleteWebTokenRequest) Check() error {
 	}
 	return nil
 }
+
+// IntoMap makes this filter into a map.
+func (f *WebSessionFilter) IntoMap() map[string]string {
+	m := make(map[string]string)
+	if f.User != "" {
+		m[keyUser] = f.User
+	}
+	return m
+}
+
+// FromMap converts provided map into this filter.
+func (f *WebSessionFilter) FromMap(m map[string]string) error {
+	for key, val := range m {
+		switch key {
+		case keyUser:
+			f.User = val
+		default:
+			return trace.BadParameter("unknown filter key %s", key)
+		}
+	}
+	return nil
+}
```

Comment:
Can you add some comments explaining how these are used? That it's paired with watcher to filter out and only return sessions for a particular user.